client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditForm.tsx

```diff
@@ -64,6 +67,16 @@ export const OutboundLineEditForm: React.FC<OutboundLineEditFormProps> = ({
   const [issueQuantity, setIssueQuantity] = useState(0);
   const { items } = useOutbound.line.rows();
 
+  const { variantsControl, asPackUnit } = useUnitVariant(
+    item?.id ?? '',
+    item?.unitName ?? null
+  );
+
+  packSizeController.options = packSizeController.options.map(option => ({
+    ...option,
+    label: option.value === -1 ? t('label.any') : asPackUnit(option.value),
```

Comment:
Only replacing the label in pack size controller so don't have to rewrite logic

Reply:
[4182c0c](https://github.com/openmsupply/open-msupply/pull/2273/commits/4182c0c1b3dfaa71e518eb7cd22c69d1b76b9fd9)